src/index.html

```diff
@@ -4,9 +4,440 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item" >
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <div class="main__container">
+        <div class="card card--margin-bottom" data-qa="card">
```

Comment:
achieve the same effect without modifier

Reply:
made  element in main block